build_automation/build_automation/urls.py

```diff
@@ -15,7 +15,11 @@
 """
 from django.contrib import admin
 from django.urls import path
+from django.conf.urls import include
+from django.conf import settings
+from django.conf.urls.static import static
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-]
+    path('content_management/', include('content_management.urls')),
```

Comment:
Since the content management's views are not ready yet, how about removing this for now?